### ISSUE TITLE
Is Stackblitz Best for Demo?

### ISSUE THREAD

**Issue by roblevintennis** [argyleink/open-props#103]
Honestly, I don't know if this is required for stackblitz to work, but it shows the following error when I visited the demo in Firefox (see righthand side)

![image](https://user-images.githubusercontent.com/142403/144749689-fd555ede-1dc5-4de6-ad4f-cf7aab77e00f.png)

Now, I know chrome is er'where, but, I'm specifically trying to use BOTH chrome and FireFox more these days because I agree with [this sentiment](https://batsov.com/articles/2021/11/28/firefox-is-the-only-alternative/) (well, to a degree).

Also, OP is really a bunch of CSS custom properties at the end of the day…why shouldn't its demo work in all modern browsers? Again, to no fault of OP code itself and I"m merely suggesting the demo tool might be better in an environment that works in another browser besides chromium. I like https://codepen.io/pen/ works maybe but probably others too.

**Comment by argyleink** [comment]
Yes, currently stackblitz doesnt work in firefox.. def not optimal. i used codesandbox at first (which does support firefox) but it's like 1/20th the speed of stackblitz. the docsite uses open props, so it's a great demo spot, i also supply multiple codepens. but the stackblitz example shows npm installation plus the use of jit-props and postcss. need a node environment to show those things, cant do codepen. 

tldr; i should add a codesandbox alternative link so folks on firefox can boot up the demo environment. or we wait for stackblitz and firefox to get along. meanwhile, there are codepen's provided that show usage of open props, where stackblitz shows npm, postcss and vite advanced optional usage. so i guess i see it as "extra" and prefer to showcase that extra in a fast tool.

**Comment by roblevintennis** [comment]
Sounds good thanks for sharing your reasons 👍🏽
